<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>go_mainwindow</class>
 <widget class="QMainWindow" name="go_mainwindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1434</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GoFigure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Resources/axes.qrc">
    <normaloff>:/fig/Myapp2.png</normaloff>:/fig/Myapp2.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="CentralTabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1434</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent_Files">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Open Recent Files</string>
     </property>
     <widget class="QMenu" name="menuSingle_Files">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Single Files</string>
      </property>
      <addaction name="separator"/>
     </widget>
     <widget class="QMenu" name="menuMultiple_Files">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Multiple Files</string>
      </property>
      <addaction name="separator"/>
     </widget>
     <addaction name="menuSingle_Files"/>
     <addaction name="menuMultiple_Files"/>
    </widget>
    <addaction name="actionOpen_Single_File"/>
    <addaction name="actionOpen_MegaCapture_Files"/>
    <addaction name="menuOpen_Recent_Files"/>
    <addaction name="separator"/>
    <addaction name="actionUse_DataBase"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Mesh"/>
    <addaction name="actionLoad_Segmentation_Results"/>
    <addaction name="separator"/>
    <addaction name="actionExport_LSM_to_MegaFile"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_all"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
    <addaction name="separator"/>
    <addaction name="actionGoFigure2_Website"/>
    <addaction name="actionUser_mailing_list"/>
    <addaction name="actionDeveloper_mailing_list"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFiltering">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Filtering</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSegmentation">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Segmentation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionAdd_Bookmark"/>
    <addaction name="actionManage_Bookmarks"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuView"/>
   <addaction name="menuFiltering"/>
   <addaction name="menuSegmentation"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="FileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Single_File"/>
   <addaction name="actionSave_Single_File_As"/>
  </widget>
  <action name="actionOpen_Single_File">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Resources/axes.qrc">
     <normaloff>:/fig/document-open.png</normaloff>:/fig/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open Single File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_MegaCapture_Files">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open MegaCapture Files</string>
   </property>
  </action>
  <action name="actionSave_Multiple_Files_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Multiple Files</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_Single_File_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../Resources/axes.qrc">
     <normaloff>:/fig/filesaveas.png</normaloff>:/fig/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save Single File As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../../Resources/axes.qrc">
     <normaloff>:/fig/system-quit.png</normaloff>:/fig/system-quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionOpen_Mesh">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Mesh / Contour</string>
   </property>
  </action>
  <action name="actionUse_DataBase">
   <property name="text">
    <string>Use DataBase</string>
   </property>
  </action>
  <action name="actionGoFigure2_Website">
   <property name="text">
    <string>GoFigure2 website</string>
   </property>
  </action>
  <action name="actionUser_mailing_list">
   <property name="text">
    <string>User mailing list</string>
   </property>
  </action>
  <action name="actionDeveloper_mailing_list">
   <property name="text">
    <string>Developer mailing list</string>
   </property>
  </action>
  <action name="actionQuadView">
   <property name="text">
    <string>QuadView XYT</string>
   </property>
  </action>
  <action name="actionFullScreen_XY">
   <property name="text">
    <string>FullScreen  XY</string>
   </property>
  </action>
  <action name="actionFull_screen_XZ">
   <property name="text">
    <string>Full screen XZ</string>
   </property>
  </action>
  <action name="actionFull_screen_YZ">
   <property name="text">
    <string>Full screen YZ</string>
   </property>
  </action>
  <action name="actionFull_screen_XYZ">
   <property name="text">
    <string>Full screen XYZ</string>
   </property>
  </action>
  <action name="actionXYT_View">
   <property name="text">
    <string>XYT View</string>
   </property>
  </action>
  <action name="actionQuadView_XYZ">
   <property name="text">
    <string>QuadView XYZ</string>
   </property>
  </action>
  <action name="actionFull_screen_XY">
   <property name="text">
    <string>Full screen XY</string>
   </property>
  </action>
  <action name="actionFull_screen_XT">
   <property name="text">
    <string>Full screen XT</string>
   </property>
  </action>
  <action name="actionFull_screen_YT">
   <property name="text">
    <string>Full screen YT</string>
   </property>
  </action>
  <action name="actionFull_screen_XYT">
   <property name="text">
    <string>Full screen XYT</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Full screen XY</string>
   </property>
  </action>
  <action name="actionSide_bar">
   <property name="text">
    <string>Side-bar</string>
   </property>
  </action>
  <action name="actionAdd_Bookmark">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Bookmark</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionManage_Bookmarks">
   <property name="text">
    <string>Manage Bookmarks</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLoad_Segmentation_Results">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Segmentation Results</string>
   </property>
  </action>
  <action name="actionExport_LSM_to_MegaFile">
   <property name="text">
    <string>Export LSM to MegaCapture</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>CentralTabWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Resources/axes.qrc"/>
 </resources>
 <connections/>
</ui>
